pkgs/by-name/an/ankama-launcher/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+  writeScriptBin,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+
+  src = fetchurl {
+    url = "https://download.ankama.com/launcher/full/linux";
+    sha256 = "693cd3411c08a4ff3bae7fc627de7ccbd9ca6e4b99d706a2d9ae283412c9edb5";
+    curlOptsList = [ "-A" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36" "-L"];
+  };
+in
+
+appimageTools.wrapType2 {
+  name = pname;
+  version = version;
+  src = src;
+  extraPkgs = pkgs: [
+    pkgs.wine
+  ];
+
+  meta = {
+    description = "Ankama Launcher";
+    longDescription = ''Ankama Launcher is a portal that lets you access all of Ankama's many universes with just a single click! Connect to our video games, VOD animations, webtoons, and even our livestreams! Download updates, stay up to date with the latest news, and chat with your friends!
+    If you have a wine error when running Dofus, delete or rename the cinematics directory. More infos on https://harbiinger.xyz/content/dofus'';
```

Comment:
```suggestion
```
Can't we delete that directory in the build script? Do we really need an external unofficial link?

Reply:
this directory will only appear if the user download the game Dofus and it can't be installed without using the launcher. Unfortunately I don't think I can automatically remove/rename it. Maybe I could add the instructions to the longDescription if it's better

Reply:
No, seems fine. I'd rather not have external unofficial links in the description, so maybe you can copy the most relevant parts of the instructions.